scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -107,6 +105,13 @@ class MicrobeEditor{
             freeBuilding = false;
         }
 
+        // Sent freebuild value to GUI
+        GenericEvent@ freebuildEvent = GenericEvent("MicrobeEditorFreeBuildToggle");
```

Comment:
I think a more suitable name for the event would be "MicrobeEditorFreeBuildStatus"

Reply:
This hasn't been renamed yet, right? (just to make sure I'm not looking at some old thing).